src/core/src/core_logic/PatchInstaller.py

```diff
@@ -177,14 +179,48 @@ def install_updates(self, maintenance_window, package_manager, simulate=False):
         self.last_still_needed_packages = all_packages
         self.last_still_needed_package_versions = all_package_versions
 
+        stopwatch_for_batch_install_process = Stopwatch(self.env_layer, self.telemetry_writer, self.composite_logger)
+        stopwatch_for_batch_install_process.start()
+
+        installed_update_count, patch_installation_successful, maintenance_window_batch_cutoff_reached = self.install_packages_in_batches(all_packages, packages, package_versions, maintenance_window, package_manager)
+
+        stopwatch_for_batch_install_process.stop()
+
+        # Check for packages available to install. After installing packages in batches, there might be still some packages available for installation. These packages should be:
+        # (a) Due to not enough remaining time in maintenance window to install the packages in batch, installing packages in batch is stopped and hence some packages are not installed in batches
+        # (b) Those which were failed while installing in batches
+        # (c) Some new packages available in this brief time when installation in batches was being done
```

Comment:
Update comment to:

 # Check for packages available and install them in batches. Some packages may not be installed due to:
        # (a) Not enough time remaining in maintenance window
        # (b) Failure during package installation
        # (c) New packages added during the brief time batch installation was in progress

Reply:
Done.